week1/homework/issues/ide.md

```diff
@@ -0,0 +1,17 @@
+## Downloading an IDE
+
+*For this assignment, you will follow the instructions and complete a task showing your knowledge of the subject at the end. If at any moment you need help, feel free to contact your TAs.*
+
+### Visual Studio Code
+
+Before we start coding, we need to install an IDE. An IDE is a software application that provides comprehensive facilities to computer programmers for software development. An IDE normally consists of at least a source code editor, build automation tools, and a debugger. Although there are hundreds of IDEs to choose from, we are going to use Visual Studio Code due to its popularity and integration with Azure (via extensions and libraries).
```

Comment:
We typically refer to VS Code as more of a "code editor" than an IDE. It's a minor distinction since you can add a lot of IDE capabilities via extensions, but folks expect a lot from an IDE that isn't available in VS Code out of the box.